swagger/generateExamples.js

```diff
@@ -0,0 +1,82 @@
+require('isomorphic-fetch')
+const fs = require('fs').promises
+const path = require('path')
+const { convert } = require('widdershins/lib/index')
+const { convertPathToFilename } = require('./helpers')
+
+const OUTPUT_DIR = path.resolve(__dirname, 'examples/')
+
+const setupData = (data) => {
+  data.operationUniqueSlug = data.method.slug
+  data.operation = data.method.operation
+  data.methodUpper = data.method.verb.toUpperCase()
+  data.url = data.utils.slashes(data.baseUrl + data.method.path)
+  data.parameters = data.operation.parameters
+  data.enums = []
+  data.utils.fakeProdCons(data)
+  data.utils.fakeBodyParameter(data)
+  data.utils.mergePathParameters(data)
+  data.utils.getParameters(data)
+}
+
+const getContentType = (operation, statusCode) => {
+  let content = operation.responses[`${statusCode}`]
+  content = content ? content.content : null
+  content = Object.keys(content || {})
+  content = content.length > 0 ? content[0] : null
+  return content
+}
+
+const fetchAndSaveExample = async (urlToFetch, filename, operation) => {
+  const res = await fetch(urlToFetch)
+  let content = await res.text()
+  const contentType = getContentType(operation, res.status)
+  if (!contentType || contentType === 'application/json') {
+    try {
```

Comment:
Nit: I like to make `try` statements as small as possible so it's clear what we think could possibly throw. Looks like that's `JSON.parse(` here?
```suggestion
    let json
    try {
        json = JSON.parse(content)
    } catch(e) {
       console.warn(...
       return
    }
    if (Array.isArray(json.data)) {
        // Shorten lists
        etc
```